utils/keycodes.js

```diff
@@ -10,3 +17,36 @@ export const DOWN = 40;
 export const DELETE = 46;
 
 export const F10 = 121;
+
+/**
+ * Check if the access keys and the given character are presssed.
+ *
+ * @param  {KeyboardEvent} event     The event object.
+ * @param  {String}        character The character to check.
+ * @return {Boolean}                 True if the combination is pressed, false if not.
+ */
+export function isAccess( event, character ) {
+	if ( isMac ) {
+		if ( ! event.ctrlKey || ! event.altKey ) {
+			return false;
+		}
+	} else if ( ! event.shiftKey || ! event.altKey ) {
+		return false;
+	}
+
+	return character ? event.keyCode === character.toUpperCase().charCodeAt( 0 ) : true;
```

Comment:
Minor: can extract `function toKeyCode( char ) { return char.toUpperCase().charCodeAt( 0 ); }`

Reply:
Readability? Or for what reason?

Reply:
Yeah, merely that. My personal preference, anyway. The function name acts as documentation, essentially.